docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,334 @@
+:::
+
+To define the `position` for a relation, pass one of the following 4 different positional attributes:
+
+| Parameter name and syntax | Description                                                            | Type       |
+| ------------------------- | ---------------------------------------------------------------------- | ---------- |
+| `before: id`              | Positions the relation before the given `id`.                          | Entry `id` |
+| `after: id`               | Positions the relation after the given `id`.                           | Entry `id` |
+| `start: true`             | Positions the relation at the start of the existing list of relations. | Boolean    |
+| `end: true`               | Positions the relation at the end of the existing list of relations.   | Boolean    |
+
+The `position` argument is optional and defaults to `position: { end: true }`.
+
+:::note Sequential order
+Since `connect` is an array, the order of operations is important as they will be treated sequentially (see combined example below).
+:::
+
+::::: tabs card
+
+:::: tab Basic example
+
+Consider the following record in the database:
+
+```json
+categories: [
+  { id: 1, order: 1 }
+  { id: 2, order: 2 }
+]
+```
+
+Sending the following request updates a `restaurant` entry and creates a third relation of `id` `3` for the `categories` attribute and position it before the relation with `id` `2`:
+
+::: request Example request to update the position of one relation
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+```json
+{
+  data: {
+    categories: {
+      connect: [
+        { id: 3, position: { before: 2 } },
+      ]
+    }
+  }
+}
+```
+
+:::
+::::
+
+:::: tab Combined example
+
+Consider the following record in the database:
+
+```json
+categories: [
+  { id: 1, order: 1 }
+  { id: 2, order: 2 }
+]
+```
+
+Sending the following example in the request body of a PUT request updates multiple relations:
+
+::: request Example request to reorder several relations
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+```json
+{
+  data: {
+    categories: {
+      connect: [
+        { id: 6, position: { after: 1} },
+        { id: 7, position: { before: 2 } },
+        { id: 8, position: { end: true } },
+        { id: 9 },
+        { id: 10, position: { start: true } },
+      ]
+    }
+  }
+}
+```
+
+:::
+
+Omitting the `position` argument defaults (as in `id: 9` defaults to `position: { end: true }`. All other relations are positioned relative to another existing `id` (using `after` or `before`) or relative to the list of relations (using `start` or `end`). Operations are treated sequentially, in the order defined in the `connect` array, so the resulting database record will be the following:
+
+```json
+categories: [
+  { id: 10, order: 1 },
+  { id: 1, order: 2 },
+  { id: 6, order: 3 },
+  { id: 7, order: 4 },
+  { id: 2, order: 5 },
+  { id: 8, order: 6 },
+  { id: 9, order: 7 }
```

Comment:
What do you think about shortening this a bit? I believe 3 / 4 items would be sufficient ...

Reply:
Thanks for the suggestion. However, I think that all of these items are important because, unless I'm missing something, they illustrate the complete result of all the operations described above:
![Screenshot 2022-12-13 at 14 10 41](https://user-images.githubusercontent.com/4233866/207326831-696888d5-5310-4bec-b201-300c617ad5cb.png)
This ordering part is a bit complex and with @Marc-Roig we discussed the importance of highlighting the different use cases. What do you think?

Reply:
There is the short example where you can see the basics of connect and disconnect syntax and then you have the more extensive one where you see all positional arguments used at once.

I would live this list with all the items, maybe what we can remove is the order attribute, which is something the API user will not see. It might seem smaller that way.

Something like:
```js
categories: [
  { id: 10 },
  { id: 1 },
  { id: 6 },
  { id: 7 },
  { id: 2 },
  { id: 8 },
  { id: 9 }
]
 
```

wdt? @pwizla @gu-stav

Reply:
I like your suggestion, Marc! WDYT, Gustav?

Reply:
Sounds good to me :)